tests/_system/test_onedim_ckd_vs_mono.py

```diff
@@ -6,10 +6,10 @@
 import pytest
 
 import eradiate
-from eradiate import unit_registry as ureg, unit_context_kernel as uck
+from eradiate import unit_context_kernel as uck
```

Comment:
This refactoring was done automatically by black, I suppose.

Reply:
By isort.

Reply:
Anyway, nothing wrong with this.